azerothcore/azerothcore-wotlk
# fix(DB/Creature): Prince Sandoval is unattackable

## CHANGES PROPOSED:
-  Update creature_template for Prince Sandoval
-  Update smart_scripts for Prince Sandoval


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3135
- Credits:  r0m1ntik94



## TESTS PERFORMED:
- Apply sql and restart server
- .tele icecrown
- .quest add 13213
- went through the quest chain 
- was able to kill Prince Sandoval at the end of the quest chain



## HOW TO TEST THE CHANGES:

- Apply sql and restart server
- .tele icecrown
- .quest add 13213
- progress through the quest chain 
- see if you can kill Prince Sandoval at the end of the quest chain

## KNOWN ISSUES AND TODO LIST:

- [x] fix invalid event flags


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
